connector_importer/components/mapper.py

```diff
@@ -53,8 +53,8 @@ def required_keys(self, create=False):
         The recordset will use this to show required fields to users.
         """
         req = dict(self.required)
-        req.update(self.work.options.mapper.get("required_keys", {}))
-        return req
+
+        return self.work.options.mapper.get("required_keys", {}) or req
```

Comment:
This looks like a breaking change to me.
```
req = {"a": "1"}
option_req = {"b": "2"}
```
before, it would've returned `{"a": "1", "b": "2"}`
Not it returns `{"b": "2"}`
What's the purpose of this ?

Reply:
The purpose is to take full control of what are the `required_keys` if we explicitly use the parameter. 
For my use case, I want to update some measurements fields over product.products where the `required_keys` and `odoo_unique_key` are barcode

It was seen during a live training session with @simahawk, but it can indeed be a subject of debate

Reply:
It's not breaking because this option is kind of new. I'm favor of this change :wink:

Reply:
I modified my commit in order to add the property required_keys_ignore_model_required into "importer.base.mapper"